custom/enikshay/case_utils.py

```diff
@@ -422,7 +422,8 @@ def _get_private_locations(person_case):
         return PrivatePersonLocationHierarchy(
             sto=state_location.metadata['nikshay_code'],
             dto=district_location.metadata['nikshay_code'],
-            pcp=pcp_location.metadata['nikshay_code'],
+            # HACK: remove this when we have all of the "HE ids" imported from Nikshay
+            pcp=pcp_location.metadata.get('nikshay_code') or None,
```

Comment:
@mkangia will this break the Nikshay integration? Instead of failing hard, I think it will attempt to send registrations even when there is no pcp Nikshay ID.

Reply:
This is fine.. the notification would fail at attempt.